api/cloud_api/util/SceGoogleApiHandler.js

```diff
@@ -41,16 +42,19 @@ class SceGoogleApiHandler {
     );
     this.oAuth2Client.setCredentials({
       // eslint-disable-next-line
-      refresh_token: REFRESH_TOKEN
+        refresh_token: REFRESH_TOKEN
     });
+    if(CLIENT_ID != 'NOT_SET' && CLIENT_SECRET != 'NOT_SET') {
```

Comment:
what if we only say they have valid api keys if theyre both set?
```suggestion
    if(CLIENT_ID != 'NOT_SET' || CLIENT_SECRET != 'NOT_SET') {
```